### PR TITLE
Ps 8.0 5427 document the correct distribution upgrade process for percona server

### PR DESCRIPTION
**Pull request by patrickbirch**

### CODE REVIEW

**Comment by dutow** [comment]
The 8.0 docs needs no change? The jira ticket mentions that 8.0 is affected.

**Comment by patrickbirch** [comment]
Updated with changes to 8.0. Thank you.

**Comment by dutow** [comment]
This is a different change than the 5.6 and 5.7 PRs, and I don't see that part in the 8.0 docs. Is this really the same issue?

File: doc/source/.res/replace.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 .. |debian| replace:: Debian
 .. |debian-last| replace:: Debian 9.0 (stretch)
+.. |debian-tenth| replace:: Debian 10.0 (buster)
```
Comment (hrvojem) [inline doc/source/.res/replace.txt:15]: if we're already changing this should we change last -> ninth?

**Review by hrvojem — CHANGES_REQUESTED** [review]
This change doesn't look to be related to distribution upgrade docs

File: doc/source/.res/replace.txt (hunk 1 above)
Comment (patrickbirch) [reply doc/source/.res/replace.txt:15]: Sure.

**Comment by patrickbirch** [comment]
duplicate pull request - closing

**Pull request closed without merging**